### ISSUE TITLE
Return to startup directory

### ISSUE THREAD

**Issue by gldanoob** [gokcehan/lf#1632]
Is there a way to return to the working directory where `lfcd` is called by a keybind? What I have tried is:
```fish
function lfcd --wraps="lf" --description="lf - Terminal file manager (changing directory on exit)"
    set -Ux old_pwd $PWD
    cd "$(command lf -print-last-dir $argv)"
end
```
and mapping a key in `lfrc`:
```bash
map u ${{cd "$old_pwd"}}
```
but that doesn't change the directory displayed by `lf`, as it runs a shell command instead of an `lf` command.

**Comment by Catalyn45** [comment]
I think one possible solution could be `map u ${{lf -remote "send ${id} cd \"$old_pwd\""}}`

**Comment by J-Kappes** [comment]
You can use the builtin environment variable `$OLDPWD`.

You can use an asynchronous shell command to reduce flickering and quit in the same mapping like this:
```
map <c-c> &{{lf -remote "send $id cd \"$OLDPWD\""; lf -remote "send $id quit"}}
```

**Comment by joelim-work** [comment]
TL;DR: Use `lf -remote "send $id cd ..."`, not `cd`:

```shell
map u &lf -remote "send $id cd \"$OLDPWD\""
```
